packages/eslint-config-eslint/default.yml

```diff
@@ -125,39 +198,41 @@ rules:
     ]
     no-return-assign: "error"
     no-script-url: "error"
-    no-self-assign: "error"
+    no-self-assign: "error" # recommended
     no-self-compare: "error"
     no-sequences: "error"
     no-shadow: "error"
     no-tabs: "error"
     no-throw-literal: "error"
     no-trailing-spaces: "error"
-    no-undef: ["error", {typeof: true}]
+    no-undef: ["error", {typeof: true}] # Modified from recommended
     no-undef-init: "error"
     no-undefined: "error"
     no-underscore-dangle: ["error", {allowAfterThis: true}]
     no-unmodified-loop-condition: "error"
     no-unneeded-ternary: "error"
     no-unreachable-loop: "error"
     no-unused-expressions: "error"
-    no-unused-vars: ["error", {vars: "all", args: "after-used", caughtErrors: "all"}]
+    no-unused-vars: ["error", {vars: "all", args: "after-used", caughtErrors: "all"}] # Modified from recommended
     no-use-before-define: "error"
     no-useless-call: "error"
     no-useless-computed-key: "error"
     no-useless-concat: "error"
     no-useless-constructor: "error"
-    no-useless-escape: "error"
+    no-useless-escape: "error" # recommended
     no-useless-rename: "error"
     no-useless-return: "error"
     no-whitespace-before-property: "error"
     no-var: "error"
+
     node/callback-return: ["error", ["cb", "callback", "next"]]
     node/handle-callback-err: ["error", "err"]
     node/no-deprecated-api: "error"
     node/no-mixed-requires: "error"
     node/no-new-require: "error"
     node/no-path-concat: "error"
-    node/no-process-exit: "error"
+    node/no-process-exit: "error" # recommended
```

Comment:
Why are recommended rules enabled here again when we already enable the recommended configs that include them?

Reply:
In the file, there are two cases where recommended rules are repeated:

1. When overriding a behavior of "warn" to change to "error"
2. Such as the example you cite where I just added the comment for those lines which can indeed just be removed if desired (I didn't know though if someone had it in mind to hard-code the rules in case the recommendations for those particular rules changed, but would be fine to remove them too.)

Reply:
I've gone ahead and added a commit to remove the rules which are already included by the recommended configs.